gpytorch/priors/prior.py

```diff
@@ -15,10 +15,9 @@ class Prior(Distribution, Module, ABC):
     respective prior in computing the Marginal Log-Likelihood.
     """
 
-    @property
-    def log_transform(self):
-        return self._log_transform
+    def transform(self, parameter):
+        return self._transform(parameter) if self._transform is not None else parameter
 
     def log_prob(self, parameter):
```

Comment:
Nit pick: could we change the argument name from `parameter`, since priors now operate on transformed parameters?

Reply:
sure I can use sth neutral like `tensor`